In-house patch to increate the maximum length of Flavor names, Flavor
keys, and Flavor values.  This patch has not yet been submitted
upstream.

--- horizon-2013.2.3/openstack_dashboard/dashboards/admin/flavors/workflows.py.orig       2014-06-27 09:55:09.196764242 -0600
+++ horizon-2013.2.3/openstack_dashboard/dashboards/admin/flavors/workflows.py        2014-06-27 09:55:12.447224496 -0600
@@ -35,7 +35,7 @@
                              "Leave this field blank or use 'auto' to set "
                              "a random UUID4.")
     name = forms.RegexField(label=_("Name"),
-                            max_length=25,
+                            max_length=255,
                             regex=r'^[\w\.\- ]+$',
                             error_messages={'invalid': _('Name may only '
                                 'contain letters, numbers, underscores, '

--- horizon-2013.2.3/openstack_dashboard/dashboards/admin/flavors/extras/forms.py.orig   2014-06-27 09:57:05.195849313 -0600
+++ horizon-2013.2.3/openstack_dashboard/dashboards/admin/flavors/extras/forms.py    2014-06-27 09:57:19.351010746 -0600
@@ -28,8 +28,8 @@
 
 
 class CreateExtraSpec(forms.SelfHandlingForm):
-    key = forms.CharField(max_length="25", label=_("Key"))
-    value = forms.CharField(max_length="25", label=_("Value"))
+    key = forms.CharField(max_length="255", label=_("Key"))
+    value = forms.CharField(max_length="255", label=_("Value"))
     flavor_id = forms.CharField(widget=forms.widgets.HiddenInput)
 
     def handle(self, request, data):
@@ -46,8 +46,8 @@
 
 
 class EditExtraSpec(forms.SelfHandlingForm):
-    key = forms.CharField(max_length="25", label=_("Key"))
-    value = forms.CharField(max_length="25", label=_("Value"))
+    key = forms.CharField(max_length="255", label=_("Key"))
+    value = forms.CharField(max_length="255", label=_("Value"))
     flavor_id = forms.CharField(widget=forms.widgets.HiddenInput)
 
     def handle(self, request, data):

